ios-theater/Base.lproj/Main.storyboard

```diff
@@ -4,20 +4,76 @@
+                                            <autoresizingMask key="autoresizingMask"/>
+                                            <subviews>
+                                                <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFill" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="banner1" translatesAutoresizingMaskIntoConstraints="NO" id="iwJ-0e-Swg">
+                                                    <rect key="frame" x="0.0" y="0.0" width="407" height="219"/>
+                                                </imageView>
+                                            </subviews>
+                                            <constraints>
+                                                <constraint firstItem="iwJ-0e-Swg" firstAttribute="top" secondItem="1o0-ab-kjy" secondAttribute="top" id="RpL-Tv-q8X"/>
+                                                <constraint firstItem="iwJ-0e-Swg" firstAttribute="leading" secondItem="1o0-ab-kjy" secondAttribute="leading" id="mjv-7a-Dxt"/>
+                                                <constraint firstAttribute="trailing" secondItem="iwJ-0e-Swg" secondAttribute="trailing" id="nJX-IY-g1P"/>
+                                                <constraint firstAttribute="bottom" secondItem="iwJ-0e-Swg" secondAttribute="bottom" id="qaE-1Y-WP4"/>
+                                            </constraints>
+                                        </collectionViewCellContentView>
+                                        <size key="customSize" width="407" height="219"/>
+                                        <connections>
+                                            <outlet property="imageView" destination="iwJ-0e-Swg" id="mKX-a4-WjI"/>
+                                        </connections>
+                                    </collectionViewCell>
+                                </cells>
+                            </collectionView>
+                            <pageControl opaque="NO" contentMode="scaleToFill" fixedFrame="YES" contentHorizontalAlignment="center" contentVerticalAlignment="center" numberOfPages="3" translatesAutoresizingMaskIntoConstraints="NO" id="cSJ-ha-6gt">
+                                <rect key="frame" x="310" y="256" width="84" height="37"/>
+                                <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                                <color key="pageIndicatorTintColor" systemColor="linkColor" red="0.0" green="0.47843137250000001" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                                <color key="currentPageIndicatorTintColor" systemColor="systemGreenColor" red="0.20392156859999999" green="0.78039215689999997" blue="0.34901960780000002" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                            </pageControl>
```

Comment:
You didn't add any constraint for `pageControl`, so it's not adaptive on other size devices.

Reply:
I'll add constrains for pageControl today.